### ISSUE TITLE
Not synchronized with mobile app

### ISSUE THREAD

**Issue by Ayad0net** [FlorianWoelki/obsidian-icon-folder#197]
**Describe the bug**
When I change the icon in desktop app not changed in the phone app

**Comment by Xtron135** [comment]
Yup me too. The plugin does not even appear in the mobile app plugin setting

**Comment by justrainer** [comment]
This is because the configuration file, that defines which folder uses which icon, is stored in the hidden `.obsidian` folder in your vault. This folder is not synced by default, though you can enable this in the plugin settings.

Be aware however that syncing your .obsidian folder will also sync the thousands of icons you have installed through the plugin, which in my own experience drastically slows down all sync operations of the plugin.
